app/models/article.rb

```diff
@@ -0,0 +1,3 @@
+class Article < ApplicationRecord
+  validates :title, presence: true, length: {minimum: 5, maximum: 78}
```

Comment:
According to a quick google search, 78 is the max length of a google email subject but perhaps a comment in the code will remind me why I chose these seemingly arbitrary numbers.